tests/Sdk/Unit/Metrics/MeterTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Tests\Sdk\Unit\Metrics;
+
+use OpenTelemetry\Sdk\Metrics\Counter;
+use OpenTelemetry\Sdk\Metrics\Meter;
+use OpenTelemetry\Sdk\Metrics\UpDownCounter;
+use OpenTelemetry\Sdk\Metrics\ValueRecorder;
+use PHPUnit\Framework\TestCase;
+
+class MeterTest extends TestCase
+{
+    public function testMeterInstrumentCreation()
+    {
+        $meter = new Meter('Meter', '0.1');
+        $this->assertInstanceOf(Meter::class, $meter);
+
+        $meter_name = $meter->getName();
```

Comment:
Looks like the codestyle is more towards camelCase other than to snake_case, no?

Reply:
Yeah we should probably make this `meterName`